src/wwwroot/controllers/TemplateCtrl.js

```diff
@@ -25,22 +26,60 @@
 
     $scope.loadInProgress = false;
     $scope.saveInProgress = false;
+    $scope.isLegacyTemplate = true;
 
-    $scope.saveHtmlRaw = saveHtmlRaw;
+    $scope.save = save;
     $scope.saveAndEditWithMseditor = saveAndEditWithMseditor;
 
     if ($routeParams["templateId"]) {
       load($routeParams["templateId"]);
     }
+    else {
+      $scope.isLegacyTemplate = !$rootScope.forceMsEditor;
```

Comment:
I do not like the name, what do you think about `$scope.isHtmlTemplate`? (because it is not legacy, is another option)

Reply:
👍